jest.config.js

```diff
@@ -0,0 +1,25 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: "coverage",
+
+  // The test environment that will be used for testing
+  testEnvironment: "node",
+
+  // A map from regular expressions to paths to transformers
+  transform: {
+    '^.+\\.tsx?$': 'ts-jest'
+  },
+
+  // A set of global variables that need to be available in all test environments.
+  globals: {
+    window: { },
+    navigator: { }, // TODO: find a way to mock the BrowserUtils
```

Comment:
Just set `navigator.userAgent` to an appropriate value?

Reply:
Found a way to mock it properly.